cpp/src/binaryop/compiled/operation.cuh

```diff
@@ -216,6 +216,34 @@ struct Pow {
   }
 };
 
+struct IntPow {
+  template <
+    typename TypeLhs,
+    typename TypeRhs,
+    std::enable_if_t<(std::is_integral_v<TypeLhs> and std::is_integral_v<TypeRhs>)>* = nullptr>
+  __device__ inline auto operator()(TypeLhs x, TypeRhs y) -> TypeLhs
```

Comment:
In the spirit of DRY, is it reasonable to modify/reuse the existing `ipow` implementation we have for `fixed_point`?

https://github.com/rapidsai/cudf/blob/de0f6cc914d4863bdc5c3c387a8eb7a46d745886/cpp/include/cudf/fixed_point/fixed_point.hpp#L78-L111

Reply:
We could try to modify/reuse but we would lose the benefit of knowing the base at compile time for the fixed point implementation.

Reply:
Ah, I should have paid closer attention to that.